static/app/components/events/interfaces/crashContent/stackTrace/content.spec.tsx

```diff
@@ -155,6 +155,68 @@ describe('StackTrace', function () {
     expect(omittedFrames).toBeInTheDocument();
   });
 
+  describe('with stacktrace improvements feature flag enabled', function () {
+    const organization = TestStubs.Organization({
+      features: ['issue-details-stacktrace-improvements'],
+    });
+
+    it('does not render non in app tags', function () {
+      const dataFrames = [...data.frames];
+      dataFrames[0] = {...dataFrames[0], inApp: false};
+
+      const newData = {
+        ...data,
+        frames: dataFrames,
+      };
+
+      renderedComponent({
+        organization,
+        data: newData,
+      });
+
+      expect(screen.queryByText('System')).not.toBeInTheDocument();
+    });
+
+    it('displays a toggle button when there is more than one non-inapp frame', function () {
```

Comment:
Looks like the show/hide behavior isn't tested. Can you add that to this test? Basically just click the button and make sure the frames are shown. Then click again and make sure they are hidden.